src/components/PopoverMenu/PopoverMenuItem.vue

```diff
@@ -26,7 +26,8 @@
 		<a v-if="item.href" :href="(item.href) ? item.href : '#' "
 			:target="(item.target) ? item.target : '' "
 			rel="noreferrer noopener" @click="action">
-			<span :class="item.icon" />
+			<span v-if="!iconIsUrl" :class="item.icon" />
```

Comment:
I would like to not have this behavior as img are supposed to be deprecated inside popovermenus and navigation :)

Reply:
The reason is that the contacts menu api of the server returns urls for the action icons. We just don't have any icon class for those.

Reply:
Then we should create the classes? :grin: